### DISCUSSION TITLE
Using Slack, is there any way of getting rid of these messages?

### DISCUSSION THREAD

**Discussion by leoheck** [ferdium/ferdium-app#1501] [Q&A]
![image](https://github.com/ferdium/ferdium-app/assets/1277920/729edf41-7919-44e7-b9f6-5ec40acd15a9)

**Comment by scbarton** [comment]
Different but related issue:  I eliminated the "Upgrade Slack" button by adding this to `recipes/slack/user.css`:

```
.p-upgrades_button { display: none !important; }
```
